index.html

```diff
@@ -180,7 +180,7 @@ <h2 class="card-header">About me</h2>
       <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </h2>
     </header>
-    <form id="contact-form" action="" target="_self" method="post">
+    <form id="contact-form" action="https://formspree.io/f/meqwrjep" target="_self" method="post">
       <fieldset>
           <input id="forminput1" type="text" name="full_name" placeholder="Full Name" required maxlength="30" />
           <label for="forminput1" class="label-remove">Full Name</label>
```

Comment:
- [ ] Kindly 😃 try to stick to the Figma Design at the styling as you see here the padding of the section contact is greater than the Figma, so please try to decrease the padding in this section to make it like the Figma Design 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/228209622-48584862-8410-4bcc-b1f5-77aaa5aeb16c.png)  |   ![image](https://user-images.githubusercontent.com/83360636/228209553-e6427074-4a08-4d71-80b0-ce5f23d6df4f.png)  |

Reply:
done sir ans for padding it is only the mobile vrsion